python/ql/lib/semmle/python/security/dataflow/ReflectedXSSCustomizations.qll

```diff
@@ -43,6 +44,15 @@ module ReflectedXss {
    */
   class RemoteFlowSourceAsSource extends Source, RemoteFlowSource { }
 
+  /**
+   * A data flow sink for "reflected cross-site scripting" vulnerabilities.
+   */
+  private class DefaultReflectedXss extends Sink {
+    DefaultReflectedXss() {
+      this = ModelOutput::getASourceNode(["html-injection", "js-injection"]).asSource()
+    }
+  }
```

Comment:
I suggest making it similar to [the javascript implementation](https://github.com/github/codeql/blob/main/javascript/ql/lib/semmle/javascript/security/dataflow/ReflectedXssCustomizations.qll#L160-L162)(also replacing source with sink):
```suggestion
  private class SinkFromModel extends Sink {
    SinkFromModel() {
      this = ModelOutput::getASinkNode(["html-injection", "js-injection"]).asSink()
    }
  }
```

Reply:
Nice spot @yoff, completely missed the `getASinkNode` 😓